### ISSUE TITLE
Search Improvement: Community/Campus partners

### ISSUE THREAD

**Issue by robynlloos** [uno-isqa-8950/uno-cpi#1386]
**Example:** When searching “The Institute for Holocaust Education” the system doesn’t pull up that there are any partners with that name. BUT that is a community partner in the system just without “The” in front of it. 
Community partner search should pull up any organization with those words within the name, and the name with the exact sequence of words entered in the search.  
![image](https://user-images.githubusercontent.com/44984230/63194450-ae34b400-c035-11e9-9154-ac7234b47fe7.png)

![image](https://user-images.githubusercontent.com/44984230/63194487-c278b100-c035-11e9-8d62-65d9a283258b.png)

![image](https://user-images.githubusercontent.com/44984230/63194522-dfad7f80-c035-11e9-9696-bc8dff20b7a4.png)

**Comment by robynlloos** [comment]
This also includes not having to re-type a new community partners name when you get beyond the search (just like with a new project name following the search). Here is the project example for reference:
![image](https://user-images.githubusercontent.com/44984230/63196247-72e8b400-c03a-11e9-9c15-f9746c128478.png)
![image](https://user-images.githubusercontent.com/44984230/63196255-75e3a480-c03a-11e9-9860-5ee720082c62.png)

**Comment by robynlloos** [comment]
The group discussed this issue in a meeting. We are looking at ways to improve the search by changing the single search bar option now into a layered search. For instance, searching by academic year, project name, and community partner to avoid duplicates.

**Comment by wlcobb** [comment]
@sirichandana95 Let's talk about this tonight.  She wants to be sure that if you type in The Institute for Holocaust Education, that Institute for Holocaust Education is displayed as a possible match.  If not, there will be duplicates in the system.  

Also, she is going to let me know if we just always search on keyword and not have a drop down for exact match.

**Comment by sirichandana95** [comment]
Sure Wendy. Thank you

**Comment by wlcobb** [comment]
@sirichandana95 @babonaresh @parimitamohanty 

The client wants to make changes to make search more intuitive for user.  Please also remember to include messages if nothing is returned etc.  So user knows what is happening.

removing the drop-downs above the search fields (i.e., “Keywords”) BECAUSE they are not intuitive, AND unnecessary because the search functionality should improve to the point that the tool is picking up any word entered if it is in any title, or project form field.  
i.	This applies to community partners and campus partners too

Please let me know if you need any more clarification.  They think that if you type in something it should look at everything and pick that information up.

**Comment by parimitamohanty** [comment]
@wlcobb I don't see the word "Keywords" anywhere.
If you find it on dev, can you send a screenshot?

**Comment by wlcobb** [comment]
@parimitamohanty It is on the search screen for partner.

![search_option](https://user-images.githubusercontent.com/42555289/67539757-c580c500-f6a8-11e9-9f85-e662ff8f230c.PNG)

**Comment by parimitamohanty** [comment]
On Thu, Oct 24, 2019 at 9:54 PM Wendy Cobb <notifications@github.com> wrote:

> @parimitamohanty
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_parimitamohanty&d=DwMCaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=vR0unsWZpPcilFTM_JdN2fKvU0NnGvnoTK_k0RkIIt4&m=93-u7nh85jiwFKbD9lji4zxR2iP8bG2F2lbks0NKOqI&s=m7breLmHy8YJUyPJ1c_Z32_W75gC7mxasD8xaYEtNMs&e=>
> It is on the search screen for partner.
>
> [image: search_option]
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__user-2Dimages.githubusercontent.com_42555289_67539757-2Dc580c500-2Df6a8-2D11e9-2D9f85-2De662ff8f230c.PNG&d=DwMCaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=vR0unsWZpPcilFTM_JdN2fKvU0NnGvnoTK_k0RkIIt4&m=93-u7nh85jiwFKbD9lji4zxR2iP8bG2F2lbks0NKOqI&s=4fzRLGwNaHshAbZxmKwlRWblsnP5lMZ3z_LXbC_u3kE&e=>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_uno-2Disqa-2D8950_uno-2Dcpi_issues_1386-3Femail-5Fsource-3Dnotifications-26email-5Ftoken-3DAKFAB4ZB4WBSC5AVPLJJZQTQQJNVHA5CNFSM4IMMLYFKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECHAQ5I-23issuecomment-2D546179189&d=DwMCaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=vR0unsWZpPcilFTM_JdN2fKvU0NnGvnoTK_k0RkIIt4&m=93-u7nh85jiwFKbD9lji4zxR2iP8bG2F2lbks0NKOqI&s=jaWVGcipU2XprcZkppHyzVOCclKDQLI4Ax_BckrXctM&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AKFAB44QDRRAT5LMBY2Z3HDQQJNVHANCNFSM4IMMLYFA&d=DwMCaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=vR0unsWZpPcilFTM_JdN2fKvU0NnGvnoTK_k0RkIIt4&m=93-u7nh85jiwFKbD9lji4zxR2iP8bG2F2lbks0NKOqI&s=6kb6P8A6sWNbPoo7NUKISKRDlnPXqu-pp-yERaEAX10&e=>
> .
>
Yes got it
Earlier I was  looking into the search project screen for it so got
confused.
All clear now.
Thanks

**Comment by parimitamohanty** [comment]
@wlcobb
No label is required above the search field right?

**Comment by wlcobb** [comment]
@parimitamohanty  Correct, eliminate the Keyword/Patrner name drop down and it should try to find best match from what type in.

**Comment by wlcobb** [comment]
@parimitamohanty @sirichandana95 The feedback is that there needs to be more context around the search.  I will get additional feedback.
